src/groups/mqb/mqbcfg/mqbcfg.xsd

```diff
@@ -97,6 +98,7 @@
       <element name='plugins'              type='tns:Plugins'/>
       <element name='messagePropertiesV2'  type='tns:MessagePropertiesV2'/>
       <element name='configureStream'      type='boolean' default='false'/>
+      <element name="tls"                  type='tns:TlsConfig' minOccurs='0'/>
```

Comment:
May want to call it `tlsConfig` or `tlsCfg`.  

More importantly, should this field go inside `networkInterfaces` or `tcpInterface`, since it is transport-related?